src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -87,7 +93,7 @@ public async Task Invoke(string githubSourceOrg, string adoSourceOrg, string git
             }
 
             var script = string.IsNullOrWhiteSpace(githubSourceOrg) ?
-                await InvokeAdo(adoSourceOrg, githubTargetOrg, ssh) :
+                await InvokeAdo(adoSourceOrg, adoTeamProject, githubTargetOrg, ssh) :
```

Comment:
We should validate that the user hasn't passed in an Ado Team Project without an Ado Org, and throw an OctoshiftCliException if they did.

Reply:
The check above says that if noth github org and ado org are empty it throws.
If both are supplied it goes the GitHub repos root, so if team project is supplied without org, then it's either missing GitHub org too and will throw or will exist